From 544f49cbe657ce04540285c67fdaa1fd3ff5db65 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Sat, 13 Sep 2014 17:03:17 +0200
Subject: [PATCH 25/61] prefs: disable hyperlink auditing

"Disable sending hyperlink auditing pings", i.e. pings to
URLs in <a ping="..."> when clicked.

The feature is already switchable through chrome://flags, so no
new UI elements need to be added.

https://html.spec.whatwg.org/multipage/semantics.html#hyperlink-auditing .
---
 chrome/browser/chrome_content_browser_client.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/chrome_content_browser_client.cc b/chrome/browser/chrome_content_browser_client.cc
index 5cdfec3a834b6..d356e4f381ea3 100644
--- a/chrome/browser/chrome_content_browser_client.cc
+++ b/chrome/browser/chrome_content_browser_client.cc
@@ -1170,7 +1170,7 @@ void ChromeContentBrowserClient::RegisterLocalStatePrefs(
 void ChromeContentBrowserClient::RegisterProfilePrefs(
     user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(prefs::kDisable3DAPIs, false);
-  registry->RegisterBooleanPref(prefs::kEnableHyperlinkAuditing, true);
+  registry->RegisterBooleanPref(prefs::kEnableHyperlinkAuditing, false);
   registry->RegisterListPref(prefs::kEnableDeprecatedWebPlatformFeatures);
   // Register user prefs for mapping SitePerProcess and IsolateOrigins in
   // user policy in addition to the same named ones in Local State (which are
-- 
2.25.1

